frontend/catalyst/compilation_pipelines.py

```diff
@@ -52,6 +52,12 @@
 jax.config.update("jax_platform_name", "cpu")
 jax.config.update("jax_array", True)
 
+try:
+    AG_AVAILABLE = True
+    from catalyst.autograph import autograph as run_autograph
```

Comment:
```suggestion
    from catalyst.autograph import autograph as run_autograph
    AG_AVAILABLE = True
```

Reply:
Just put it this way because the formatter inserts a new line after the import 😅 but you're right this would be safer 👍

Reply:
https://github.com/PennyLaneAI/catalyst/pull/235/commits/01de9c315b08144d08c3a81b2e44c8bb3da682c4